metacatalog/db/revisions/rev14.py

```diff
@@ -0,0 +1,42 @@
+"""
+Metacatalog database revision
+-----------------------------
+date: 2023-01-27T17:10:15.164557
+
+revision #14
+
+Drop deprecated column entries.geom
+
+"""
+from sqlalchemy.orm import Session
+from metacatalog import api, models
+
+UPGRADE_SQL = """
+-- drop column entries.geom
+ALTER TABLE entries
+DROP COLUMN geom;
+COMMIT;
+"""
+
+DOWNGRADE_SQL = """
+-- add column entry.geom
+ALTER TABLE entries
+ADD geom geometry(Geometry,4326);
+COMMIT;
+"""
+
+# define the upgrade function
+def upgrade(session: Session):
+    print('run update')
+    # drop column entries.geom
+    with session.bind.connect() as con:
+        con.execute(UPGRADE_SQL)
```

Comment:
should we already use the SQLAlchemy 2.0 syntax here (`text`) and so on?

maybe not, because `session.bind` won't exist then...

Reply:
That would be better, true.

I would change the code of all revisions in the future when we know how to deal with the fact that `session.bind` is no longer available.

Reply:
Ok, but then, let's have a migration of all revisions to the SQLAlchemy 2.0 syntax at one point and keep the old syntax for now.